src/mame/drivers/galivan.cpp

```diff
@@ -108,7 +108,11 @@ void dangarj_state::dangarj_io_map(address_map &map)
 
 WRITE8_MEMBER(galivan_state::blit_trigger_w)
 {
-	m_nb1414m4->exec((m_videoram[0] << 8) | (m_videoram[1] & 0xff),m_videoram,m_scrollx,m_scrolly,m_tx_tilemap);
+	uint16_t message = (m_videoram[0] << 8) | (m_videoram[1] & 0xff);
+	// For avoid corrupt of the continue screen, consecutived screen clear command (0x2XX) is ignored.
+	if ((message & 0xff00) != 0x200 || message != m_ninjemak_prevmessage)
+		m_nb1414m4->exec(message, m_videoram, m_scrollx, m_scrolly, m_tx_tilemap);
+	m_ninjemak_prevmessage = message;
```

Comment:
Do you have any evidence that it really behaves like this?  Only ignoring consecutive clear commands looks a lot like a hack to work around some other problem.

Reply:
In continue screen of ninjak, first send screen clear command, and after drawing "CONTINUE PLAY?", it sends the same command again immediately.
If at least the second screen clear command functions normally, the continue screen will be currupted.
While the NB1414M2 is processing the screen clear command, it is based on the assumption that the new screen clear command will be ignored.
However, it is certainly a hack-like implementation, so I try change it to a better one.

Reply:
NB1414M4 is a funny chip, and it is unknown if it's an exotic DMA/blitter, a gate array or a MCU. 
That said, it doesn't seem impossible to me that they have some kind of command locking if same trigger happens.

Reply:
I tried to use a timer to ignore quick commands.
But it was also done on the Terraforce title screen. They are a different command code and both must be processed successfully.
Maybe the trigger is the same command, not a quick command.

Reply:
To improve hack-like handling code, the command ignoring routine has been moved from the driver (galivan.cpp) to the device (nb1414m4.cpp).

Reply:
It has been resolved because it was split into another pull request #6070.